packages/text-input-react/src/BaseInputField.tsx

```diff
@@ -40,11 +40,11 @@ function getWidthAsStyle(width?: string, maxLength?: number): CSSProperties | un
 }
 
 export const BaseInputField = forwardRef<HTMLInputElement, Props>(
-    ({ id, describedBy, invalid, maxLength, width, type = "text", ...rest }, ref) => (
+    ({ id, describedBy, invalid, maxLength, width, type = "text", className, ...rest }, ref) => (
         <input
             ref={ref}
             id={id}
-            className="jkl-text-input__input"
+            className={`jkl-text-input__input ${className}`}
```

Comment:
Bør nok være en ternary for `className` her, ellers vil du ende opp med "jkl-text-input__input undefined" når ikke `className` er gitt.

Reply:
Godt poeng, tenkte det var så liten task så glemte å tenke skikkelig 🤦‍♂️